### PR TITLE
Port functionality #73

### LINKED ISSUE #73: Port functionality from old frontend to new one
**THIS TASK/ISSUE DEPENDS ON #72** 
After the task/issue #72 has been completed, reviewed, approved and merged, this task should be done ASAP.
The task/issue includes:
- porting the most important and (for us) relevant functionality from the old frontend project to the new one.
- testing the functionality and connection to the backend.

### PR DESCRIPTION
**Pull request by dietrich14**
resolves #73 
ported the most important components, further adaption will be needed when designing the dashboard as well as the admin component.

### CODE REVIEW

**Comment by amarbajric** [comment]
Correct me if I am wrong, but it seems like you have only copied the most relevant components from the old frontend to the new one. The code however looks the same and has not been modified or refactored by you?
While this is acceptable in the beginning, it will be a lot of work later on if we actually need these components. The logic of the components (`.ts`), the services and also the views (`.html`, `.scss`) will need a huge refactor in order to fit into the new frontend.

**TL;DR:**
We can merge it into the dev as it provides a lot of components which can be reused.
Keep in mind however, that we will need to put a lot of effort into it for refactoring.

@tortmann do you agree with me or do you have any other opinion?

**Comment by tortmann** [comment]
I definitely agree that refactoring will be a lot of work. However the question is whether or not we want to refactor everything or only the components / code as soon as it is required for new feature implementations.


Merging is approved for now though

**Comment by amarbajric** [comment]
We will definitely not refactor everything from the old components as I highly doubt that we will need every single component.
Nevertheless, refactoring will be necessary in either way (removing unnecessary components or code and refactoring components that are needed).

**Review by amarbajric — DISMISSED** [review]

**Review by amarbajric — APPROVED** [review]
